index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <nav>
+                <div class="left-bar">
+                    <p class="time">9:41</p>
+                    <p class="name">Oscar Byiringiro</p>
```

Comment:
- [ ] Your logo should use a link (`a` tag). * where the `href` attribute will link users to home page. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- Please replace this `p` tag with an `a` tag.